c10/core/Allocator.h

```diff
@@ -228,10 +228,20 @@ struct C10_API MemoryReportingInfoBase : public c10::DebugInfoBase {
   MemoryReportingInfoBase();
   virtual ~MemoryReportingInfoBase() {}
 
-  // Negative alloc_size corresponds to freeing of the memory
+  /**
+   * alloc corresponds to the size of the ptr
+   *
+   * allocated corresponds to total allocated memory (e.g. from allocator to
```

Comment:
"from allocator to tensor" is a confusing comment, better delete it.

Reply:
updated